pkg/cmd/roachprod-microbench/compare.go

```diff
@@ -122,7 +171,9 @@ func (c *compare) publishToGoogleSheets(
 		if c.sheetDesc != "" {
 			sheetName = fmt.Sprintf("%s (%s)", sheetName, c.sheetDesc)
 		}
-		url, err := c.service.CreateSheet(c.ctx, sheetName, *metricMap, "old", "new")
+
+		comparisonMap := c.createComparisons(*metricMap, "old", "new")
```

Comment:
Nit: since we have two functions that utilise this, we could lift this out and pass it to `publishToGoogleSheets`  and `compareUsingThreshold`.